src/secrets.js

```diff
@@ -180,4 +200,36 @@ function exportSecretToOutput(variableName, secretValue, exportSecretsToOutputs,
     }
 }
 
+function exportJsonFields(parsedJson, secretName, prefix, exportSecretsToOutputs, exportSecretsToEnvironment) {
+    if (prefix === undefined) {
+        prefix = convertPathNameToPrefix(secretName)
+    }
+    let outputName;
+    for (let key in parsedJson) {
+        outputName = prefix + "_" + key.toUpperCase()
+        exportSecretToOutput(outputName, parsedJson[key], exportSecretsToOutputs, exportSecretsToEnvironment)
+    }
+}
+
+function checkDuplicateKeys(parsedJson) {
```

Comment:
This is a bad function name imo. Implies it doesn't have side effects, but it does.

Reply:
Changed